public/css/login.css

```diff
@@ -0,0 +1,261 @@
+body {
+    background-color: #dfdfdf;
+}
+
+/* Universal Selector */
+* {
+    
+}
+
+a{
+    color: #FFFFFF;
+}
+
+a:visited{
+    color: #FFFFFF;
+}
+
+.ShakeGuard-Logo{
+    margin-top: 5vh;
+
+    width: 50vw;
+    height: auto;
+}
+
+#Base-Container{
+    display:grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+}
+
+#Base-Container-Image{
+    z-index: 0;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+    background-image: url(../images/mountain1.jpg);
+    filter: blur(8px);
+    -webkit-filter: blur(1.5px);
+    margin-left: 5vw;
+    margin-right: 5vw;
+    margin-top: 5vh;
+    margin-bottom: 5vh;
+    border-radius: 25px;
+
+    grid-column: 1 / span 1;
+    grid-row: 1 / span 1;
+}
+
+#Base-Container-OpacityCard{
+    z-index: 1;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+    /* background-image: url(../images/mountain1.jpg); */
+    background-color: #17A5BA; 
+    opacity: .75;
+    margin-left: 5vw;
+    margin-right: 5vw;
+    margin-top: 5vh;
+    margin-bottom: 5vh;
+    border-radius: 25px;
+
+    grid-column: 1 / span 1;
+    grid-row: 1 / span 1;
+}
+
+#Form-Container{
+    z-index: 2;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+    /* background-color: #d2d2d2; */
+    margin-left: 5vw;
+    margin-right: 5vw;
+    margin-top: 5vh;
+    margin-bottom: 5vh;
+    border-radius: 25px;
+    box-shadow: 5px 5px 10px rgb(179, 179, 179);
+    grid-column: 1 / span 1;
+    grid-row: 1 / span 1;
+    
+}
+
+.Grid-Container-Form-Input{
+    background-color: #FFFFFF;
+    text-indent: 10px;
+    margin: 10px 0;
+    border: none;  
+    border-radius: 25px;
+    width: 70vw;
+    height: auto;
+}
+
+#Button-Login{
+    border: none;    
+    border-radius: 25px;
+    background-color: #FBB33B;
```

Comment:
Might be worth putting commonly-used colours in [CSS Custom Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).